### DISCUSSION TITLE
Frigate plus annotation categories

### DISCUSSION THREAD

**Discussion by matt-snelling** [blakeblackshear/frigate#5068] [General]
Hello,

Couple of things:

Firstly, Frigate is great  - I was overhauling my stuff over Christmas and I won't be renewing BlueIris.  You can have that money instead when "Plus" goes live.

First thing:
I've noticed a disparity between Frigate labels / annotation and the main Frigate AI model labels.  I can get Frigate to detect bicycles - something I'm very keen for it to do, as I was burgled a few years ago and that was what prompted me to install CCTV.

I cannot however, train your upcoming Frigate+ AI model with bicycles, as the category is missing:
![image](https://user-images.githubusercontent.com/14086708/212335780-0d191cd5-e80a-4499-92fc-dc9c518af95e.png)

Second thing:
perhaps we move away from "ups" "fedex" "amazon" etc, and perhaps just go with a generic "courier" because these are perhaps, really quite country specific - once there is enough submissions, I wouldn't have thought it would be too troublesome for the model to detect, generally speaking, what a courier looks like - even if it was, "DHL" or "DPD" for example.  End of the day, Couriers do have a "general look" don't they?  A van with some bright insignia / logos.  This could of course start triggering false positives with trades, like heating engineer or electricians...  best route forward needs further discussion.

An alternative though, is to be "specific" and then you're going to get users asking for their list of couriers to be added, and that's going to get big quickly!

**Comment by NickM-27** [comment]
> I cannot however, train your upcoming Frigate+ AI model with bicycles, as the category is missing:

You have to start somewhere. In the main discussion https://github.com/blakeblackshear/frigate/discussions/1043 it has already been said that more labels are coming.
